dti-website-redesign/src/pages/products/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { Card } from 'react-bootstrap';
+import Layout from '../../components/Layout';
+import styles from './index.module.css';
+
+type ProductCard = {
+  imageURL: string;
+  siteURL: string;
+  year: string;
+  name: string;
+  description: string;
+};
+
+const productCards: ProductCard[] = [
+  {
+    imageURL: '/static/products/cureview.png',
+    siteURL: '/products/cureviews',
+    year: 'Fall 2017',
+    name: 'CU Reviews',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/queuemein.png',
+    siteURL: '/products/queuemein',
+    year: 'Fall 2017',
+    name: 'Queue Me In',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/courseplan.png',
+    siteURL: '/products/courseplan',
+    year: 'Fall 2019',
+    name: 'CoursePlan',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/carriage.png',
+    siteURL: '/products/carriage',
+    year: 'Fall 2019',
+    name: 'Carriage',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/cuapt.png',
+    siteURL: '/products/cuapt',
+    year: 'Fall 2020',
+    name: 'CU Apartments',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/designatcornell.png',
+    siteURL: '/products/designatcornell',
+    year: 'Fall 2020',
+    name: 'Design @ Cornell',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/zing.png',
+    siteURL: '/products/zing',
+    year: 'Spring 2021',
+    name: 'Zing',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  },
+  {
+    imageURL: '/static/products/cornellgo.png',
+    siteURL: '/products/cornellgo',
+    year: 'Fall 2021',
+    name: 'Cornell Go',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  }
+];
```

Comment:
This data should probably be put into a `.json` file somewhere to make it easier for people to update product info

Reply:
Agreed, done.